doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -2809,22 +2812,27 @@ Configure a {product-title} instance to act as a remote copy where data from the
 [[configuring-the-global-copy]]
 ===== Configuring the Global Copy
 
-Configure a {product-title} instance to act as the global copy from which data is replicated to remote copies. This procedure must only be performed on servers where the *Database Synchronization* role will be enabled. These settings must be completed before enabling that role.
+Configure a {product-title} instance to act as the global copy from which data is replicated to remote copies.
 
-. Navigate to menu:Settings[Configuration].
+image:centralized_admin_enable.png[Enable Centralized Administration]
+
+. Navigate to the user settings in the top-right menu.
+. Click *Configuration*.
 . Click the *Settings* accordion.
 . Click the region where the instance is located.
 . Click *Replication*.
 . Select *Global* from the *Type* list.
-. Add a subscription:
-.. Click *Add Subscrption*.
+. Click *Add Subscription*.
 .. Enter the name of the database on the remote copy in the *Database* field.
 .. Enter the IP address or fully qualified domain name of the remote copy in the *Host* field.
 .. Enter the name of a database user able to access the database in the *Username* field.
 .. Enter the password of the database user in the *Password* field.
 .. Enter the port by which the database is accessed in the *Port* field.
+.. In *Central Admin Enabled*, select *Yes* to enable central administration. 
```

Comment:
This is not correct. Central Administration can only be enabled after the initial data sync is completed. So the subscription has to be saved before this can be done.

Also, this is not enabled through the switch on the UI, but from the menu on the right of the subscription row.